source/common/http/conn_manager_impl.h

```diff
@@ -120,48 +130,72 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     void commonBufferDataIfStopAll(Buffer::Instance& provided_data, bool& buffer_was_streaming);
 
     void commonContinue();
+
     virtual bool canContinue() PURE;
+
     virtual Buffer::WatermarkBufferPtr createBuffer() PURE;
+
```

Comment:
Please don't add extra blank space here, this PR is already very large and these aren't essential changes. In general, Google/Envoy style encourages dense packing unless i'ts helpful for readability.